### DISCUSSION TITLE
antd upload 抽离组建无法获取到值 ,也监听不到触发

### DISCUSSION THREAD

**Discussion by Julycc41** [ant-design/ant-design#41664] [Q&A]
复现地址https://codesandbox.io/s/xiao-yan-qi-ta-zu-jian-antd-5-4-0-forked-6syfdx?file=/demo.tsx:873-879

**Comment by Julycc41** [comment]
getValueFromEvent={normFile}。获取不到。保存form 也获取不到

**Comment by zombieJ** [comment]
你的 UploadComponent 没有把事件透传，Form 自然就不知道内部状态了。参考一下自定义示例：https://ant.design/components/form-cn#components-form-demo-customized-form-controls
